### DISCUSSION TITLE
Compiti con data

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#639] [Idee e proposte]
Sarebbe possibile nella versione successiva inserire la possibilita di specificare la data di consegna dei compiti assegnati?grazie

**Comment by trinko** [comment]
Scusa, ma è già così: la data di consegna è quella che puoi selezionare quando inserisci un compito nell'agenda.
La data di inserimento è invece registrata in automatico.
